androidsdk/src/main/java/com/wootric/androidsdk/objects/LocalizedTexts.java

```diff
@@ -136,16 +138,34 @@ public LocalizedTexts createFromParcel(Parcel source) {
         }
     };
 
-    public static LocalizedTexts fromJson(JSONObject localizedTextsJson) throws JSONException {
+    public static LocalizedTexts fromJson(JSONObject localizedTextsJson, String surveyType) throws JSONException {
         LocalizedTexts localizedTexts = new LocalizedTexts();
-        localizedTexts.npsQuestion = localizedTextsJson.optString("nps_question");
-
-        JSONObject anchorsJson = localizedTextsJson.optJSONObject("anchors");
-        localizedTexts.anchors = new HashMap<>();
 
-        if(anchorsJson != null) {
-            localizedTexts.anchors.put(ANCHOR_LIKELY_KEY, anchorsJson.optString(ANCHOR_LIKELY_KEY));
-            localizedTexts.anchors.put(ANCHOR_NOT_LIKELY_KEY, anchorsJson.optString(ANCHOR_NOT_LIKELY_KEY));
+        if (surveyType.equals(Constants.CES) || surveyType.equals(Constants.CSAT)) {
```

Comment:
Nested `if` statements can mean the need for a function, what do you think? up to you...